app/components/provider_interface/language_skills_component.rb

```diff
@@ -34,14 +34,14 @@ def language_details_row
     def english_main_language_details_row
       {
         key: t('application_form.personal_details.english_main_language_details.label'),
```

Comment:
Looks like these needed updating too?

Reply:
@fofr I must admit I find this a bit confusing but it looks right to me - the label here is _Other languages spoken_ which works if English main language is 'Yes'. I checked it locally and I see this:

![image](https://user-images.githubusercontent.com/450843/69445089-8ac28900-0d49-11ea-9692-506486afb884.png)